### ISSUE TITLE
Tracktype for bridges

### ISSUE THREAD

**Issue by fxedel** [streetcomplete/StreetComplete#5566]
Paths with `highway=track` may contain bridges, see here: https://www.openstreetmap.org/way/122304106

For these bridges, the tracktype quest (#487) „What’s the surface firmness of this track?“ is a bit misleading, since the icon and the images don't really fit to bridges.

I don't know what's the best way to fix this, some ideas:
* Hide quest for bridge=*
* Change quest wording / special quest for bridges
* Such bridges are incorrectly mapped and should actually not be `highway=track`

**Versions affected**
v57.1

**Comment by fxedel** [comment]
> Why do you think that a bridge cannot be a highway=track? What highway value would you give to a bridge that is mainly used to access an agricultural area?

I was just unsure _if_ this is incorrectly mapped. Apparently, it's not.

I don't really know how to answer the tracktype question for example for wooden bridges. I would probably go with grade 1, since wood is solid, but I wouldn't say it's as solid as asphalt. The [tracktype wiki page](https://wiki.openstreetmap.org/wiki/Key:tracktype) gives me the impression that this key is not meant for wooden surfaces. The images from StreetComplete only cover gravel/compacted/dirt-like surfaces, none of them fits to a wooden surface.

So maybe the confusion comes mostly from combining a "solid" surface with tracktype?

**Comment by matkoniecz** [comment]
> I would probably go with grade 1, since wood is solid

I agree and I tag it this way

> The images from StreetComplete only cover gravel/compacted/dirt-like surfaces, none of them fits to a wooden surface.

I guess if someone would find set of photos for bridges we could have them? But that would be a loooot of work - photos need to be in matching perspective, with surface as focus, no distracting elements, space at the bottom... Switching only some with dedicated bridge images would increase confusion.

And so far only case of reported confusion resulted in tagging as it should be done.

So for now I would treat it as being answered, with confusion hopefully cleared - and will close this issue.

**Comment by kmpoppe** [comment]
StreetComplete has created a total of `175094` notes, of which `448` are for the quest "What’s the surface firmness of this track?" (0.256%). `55` of these notes (12.28%) are for ways that are tagged with `highway=track`+`bridge=yes` (see [this https://gist.github.com/kmpoppe/8b201b777f12f980d25ff2e173c84c95] list).

After looking through half of them, all bar 2 wrote something like "this is a bridge with this-and-that surface".

I would very much think that there is more confusion than just one instance and would therefore, as @fxedel suggested, ignore the quest for ways with `bridge=*`.

K

**Comment by mnalis** [comment]
**TL;DR:** yes, tracktype quest could be skipped on bridges, as it is confusing, contributes little, and often seems mistagged anyway. I'd recommend making a PR @kmpoppe and see how it goes.

---

Reasoning:

- I do agree that tracktype on bridges can be confusing, and this being StreetComplete, it would make sense to skip them for that reason alone.

- since bridges are usually very small (compared to rest of the ways one has to travel) and usually at least the same (or better) then tracktype of surrounding track, skipping asking that particular quest on bridges won't ignore valuable data.

- for details of bridges, `surface=*` (material) and `smoothness=*` are much more important.

- I'm not sure if `tracktype` on `bridge` is correctly tagged in most cases anyway; cursory look would seem to suggest it is more likely people often just leave existing tags (including `tracktype`) of surrounding track when splitting the way to add a bridge. 
  I cannot explain otherwise [14k+ grade5](https://taginfo.openstreetmap.org/tags/tracktype=grade5#combinations) surface firmness bridges (almost the fifth of the [grade1 bridges](https://taginfo.openstreetmap.org/tags/tracktype=grade1#combinations)!)... Even if it has soil over it, it is **still firm** - i.e. your vehicle wheels won't drop into into even after heavy rain (unless it's base is completely rotten wood and unsafe to use, in which case it should be `abandoned:highway` anyway at least). As the [wiki](https://wiki.openstreetmap.org/wiki/Key:tracktype) says: _"Tracktype is a measure of how well-maintained a track or other road is, particularly regarding surface firmness."_

**Comment by kmpoppe** [comment]
@mnalis Thanks for reopening this, I will file a PR. Might take a day or two. 

Thx, K.
